js/src/pages/price-benchmark/price-benchmark-suggestions.js

```diff
@@ -1,63 +1,120 @@
+		),
+		label: LABELS[ LABEL_PRICE_CHANGE_EFFECTIVENESS ].title,
+		render: ( { item } ) => {
+			return (
+				<EffectivenessIndicator
+					effectiveness={ item[ 'price-change-effectiveness' ] }
+				/>
+			);
+		},
 	},
 	{
-		key: 'effectiveness',
-		label: __( 'Price Change Effectiveness', 'google-listings-and-ads' ),
-		isLeftAligned: true,
-		required: true,
+		id: 'regular-price',
+		enableHiding: false,
+		enableSorting: true,
+		enableGlobalSearch: false,
+		label: <Label labelKey={ LABEL_REGULAR_PRICE } />,
+		render: ( { item } ) => {
+			return <Price amount={ item[ 'regular-price' ] } highlight />;
+		},
 	},
 	{
-		key: 'regular-price',
-		label: __( 'Regular Price', 'google-listings-and-ads' ),
-		isLeftAligned: true,
-		required: true,
+		id: 'price-on-google',
+		enableHiding: false,
+		enableSorting: true,
+		enableGlobalSearch: false,
+		header: <Label labelKey={ LABEL_PRICE_ON_GOOGLE } />,
+		label: LABELS[ LABEL_PRICE_ON_GOOGLE ].title,
+		render: ( { item } ) => {
+			return <Price amount={ item[ 'price-on-google' ] } />;
+		},
 	},
 	{
-		key: 'avg-google-price',
-		label: __( 'Avg. Price on Google', 'google-listings-and-ads' ),
-		isLeftAligned: true,
-		required: true,
+		id: 'price-gap',
+		enableHiding: false,
+		enableSorting: true,
+		enableGlobalSearch: false,
+		header: <Label labelKey={ LABEL_PRICE_GAP } />,
+		label: LABELS[ LABEL_PRICE_GAP ].title,
+		render: ( { item } ) => {
+			return `${ item[ 'price-gap' ] }%`;
+		},
 	},
 	{
-		key: 'price-gap',
-		label: __( 'Price Gap', 'google-listings-and-ads' ),
-		isLeftAligned: true,
-		required: true,
+		id: 'suggested-price',
+		enableHiding: false,
+		enableSorting: true,
+		enableGlobalSearch: false,
+		header: <Label labelKey={ LABEL_SUGGESTED_PRICE } />,
+		label: LABELS[ LABEL_SUGGESTED_PRICE ].title,
+		render: ( { item } ) => {
+			return <Price amount={ item[ 'suggested-price' ] } />;
+		},
 	},
 	{
-		key: 'suggested-price',
-		label: __( 'Suggested Price', 'google-listings-and-ads' ),
-		isLeftAligned: true,
-		required: true,
-	},
-	{
-		key: 'actions',
-		label: __( 'Action', 'google-listings-and-ads' ),
-		isLeftAligned: true,
-		required: true,
+		id: 'action',
+		enableHiding: false,
+		enableSorting: false,
+		enableGlobalSearch: false,
+		label: LABELS[ LABEL_ACTION ].title,
+		render: ( { item } ) => {
+			return <ChangePrice id={ item.id } />;
+		},
 	},
 ];
 
+const TABLE_FIELDS_MOBILE = [ 'action' ];
+
+/**
+ * PriceBenchmarkSuggestions component.
+ *
+ * This component fetches and displays price benchmark suggestions using the
+ * `usePriceBenchmarkSuggestions` hook. It renders a table with the suggestions
+ * data and handles responsiveness by providing different field configurations
+ * for desktop and mobile views.
+ *
+ * @return {JSX.Element} A div containing the PriceBenchmarkTable component.
+ */
 const PriceBenchmarkSuggestions = () => {
+	const { suggestions, hasFinishedResolution } =
+		usePriceBenchmarkSuggestions();
```

Comment:
Should we set a default value here?

```suggestion
	const { suggestions = [], hasFinishedResolution } =
		usePriceBenchmarkSuggestions();
```

Reply:
@joemcgill I feel, in that case, it's best we set the default value in `PriceBenchmarkTable` which I just did. Then when we need to render the `EmptyMetricsNotice` component, we can check if `data.length === 0` and `isReady === true` in `PriceBenchmarkTable`